### PR TITLE
refactor(jobs/tikv/pd): refactor job pd_ghpr_build

### PR DESCRIPTION
**Pull request by wuhuizuo**
current will it will be triggered manually

### CODE REVIEW

File: pipelines/tikv/pd/latest/pod-ghpr_build.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+apiVersion: v1
+kind: Pod
+spec:
+  securityContext:
+    fsGroup: 1000
+  containers:
+    - name: golang
+      image: "hub.pingcap.net/jenkins/centos7_golang-1.18:latest"
```
Comment (purelind) [inline pipelines/tikv/pd/latest/pod-ghpr_build.yaml:8]: need use go1.19 image 'hub.pingcap.net/jenkins/centos7_golang-1.19:latest'

Comment (wuhuizuo) [reply pipelines/tikv/pd/latest/pod-ghpr_build.yaml:8]: old is still go-1.18

Comment (wuhuizuo) [reply pipelines/tikv/pd/latest/pod-ghpr_build.yaml]: fixed

**Review by purelind — APPROVED** [review]

**Pull request merged**